### PR TITLE
Update some obsolete URLs, and shorten others

### PR DESCRIPTION
**Pull request by ctrueden**
TTSIA.

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

File: components/bio-formats-plugins/src/loci/plugins/About.java [hunk 1]
```diff
@@ -44,11 +44,11 @@
 
   /** URL of the Bio-Formats web page. */
   public static final String URL_SOFTWARE =
-    "http://www.openmicroscopy.org/site/products/bio-formats";
+    "http://openmicroscopy.org/info/bio-formats";
```
Comment (sbesson) [inline components/bio-formats-plugins/src/loci/plugins/About.java]: No strong veto to this change that but a mid-term implication is that this info redirect will need to be actively tested/maintained. /cc @hflynn

File: components/bio-formats-plugins/src/loci/plugins/About.java [hunk 2]
```diff
@@ -44,11 +44,11 @@
...
 
   /** URL of Bio-Formats ImageJ web page. */
   public static final String URL_BIO_FORMATS_IMAGEJ =
-    "http://www.openmicroscopy.org/site/support/bio-formats/users/imagej/features.html";
+    "http://openmicroscopy.org/info/bio-formats/users/imagej/features.html";
```
Comment (qidane) [inline components/bio-formats-plugins/src/loci/plugins/About.java]: This is not how the info redirects were meant to work. The initial use was `http://openmicroscopy.org/info/bio-formats` and that plus anything below that should redirect to `http://www.openmicroscopy.org/site/products/bio-formats`.

This means that `http://openmicroscopy.org/info/bio-formats/any-sub-path` is not supported and is probably only working by luck.

**Comment by qidane** [comment]
We have only used the info URLs on leaflets and posters, where we wanted to keep them short as people would be typing them in.

**Comment by hflynn** [comment]
Yeah, the info redirects are not intended for this sort of use. The Data Model doc links are an improvement though, thanks. @ctrueden can you roll back the 2nd commit or would it be easier for me to open a new PR with just that commit for you?

**Comment by hflynn** [comment]
Can someone close this please? I've cherry-picked the first commit to open a new PR. Thanks again @ctrueden

**Comment by ctrueden** [comment]
> This is not how the info redirects were meant to work.

Very sorry to hear that.

> We have only used the info URLs on leaflets and posters

I have linked to `info` subpaths many times, which are now embedded in many list archive messages. Just in 2015 alone:

* https://list.nih.gov/cgi-bin/wa.exe?A2=IMAGEJ;5f97c1d1.1503
* https://groups.google.com/d/msg/icy-software/CBw0QCTyv6Q/nPGUgySUnNsJ
* https://groups.google.com/d/msg/fiji-devel/RnVgeAYfBZY/sOSNpcmM2wwJ
* https://list.nih.gov/cgi-bin/wa.exe?A2=IMAGEJ;c13c0558.1501

And many more before that.

> This means that `http://openmicroscopy.org/info/bio-formats/any-sub-path` is not supported and is probably only working by luck.

I urge you not to "fix" this behavior. It will break all the previous links, for no pragmatic purpose whatsoever.

**Comment by ctrueden** [comment]
@qidane Here is an excerpt from a mail you wrote to `ome-nitpick` on Aug 30, 2013 with subject "URL shortcut redirects":

> Links in the following form will now work:
> http://openmicroscopy.org/info/bio-formats/developers/non-java-code.html (or any other sub folder)
> http://openmicroscopy.org/info/bio-formats/
> http://openmicroscopy.org/info/bio-formats

So it seems that originally, the `info` links _were_ intended to work that way.

**Comment by qidane** [comment]
Nope, no memory of that at all. I guess I must have reworked them for some reason... wonder why. It seems unlikely we would have put the long form on a poster or leaflet.

**Comment by ctrueden** [comment]
@qidane If you read that nitpick thread, you recommended the `info` links as a way of giving the shortest possible links to people in mailing list replies, etc. Do you want me to stop doing that? If so, what is the alternative?

**Comment by qidane** [comment]
I think it is a decision for @hflynn how you want to handle this going forward.

**Comment by hflynn** [comment]
Links in an email can be pasted into a browser or clicked directly so I don't see it matters how long the url is - best just to use the real link. The original point of the info redirects was because people have to type urls from leaflets or posters so shorter/easier to remember has value.

**Comment by ctrueden** [comment]
@hflynn Nonetheless, I hope the `info` links continue to work for the foreseeable future. They are very useful beyond that use case. And there are many uses of them in the wild now.

**Pull request closed without merging**